src/Agent.Worker/Release/ReleaseFileSystemManager.cs

```diff
@@ -55,6 +57,17 @@ public StreamReader GetFileReader(string filePath)
 
             return new StreamReader(new FileStream(path, FileMode.Open, FileAccess.Read, FileShare.Read, StreamBufferSize, true));
         }
+        
+        public FileInfo GetFileInfo(string filePath)
+        {
+            string path = Path.Combine(ValidatePath(filePath));
+            if (!File.Exists(path))
+            {
+                throw new FileNotFoundException(StringUtil.Loc("FileNotFound", path));
+            }
+
+            return new FileInfo(filePath);
```

Comment:
not sure if just constructing the FileInfo object with the file path is good enough to retrieve the info of the file. are you sure?